ocaml/idl/datamodel_errors.ml

```diff
@@ -107,6 +107,12 @@ let _ =
     ~doc:"A VDI with the specified location already exists within the SR" ();
   error Api_errors.memory_constraint_violation ["constraint"]
     ~doc:"The dynamic memory range does not satisfy the following constraint." ();
+  error Api_errors.memory_constraint_violation_order ["constraint"]
+    ~doc:"The dynamic memory range violates static_min <= dynamic_min <=
```

Comment:
Are you sure you want to have a newline in the string here? I think that'll translate into a newline in the error message which may not be what we want (I think XenCenter only displays a single line)